City-of-Helsinki/drupal-hdbt
# UHF-10772: Extend linkedevents app

# [UHF-10772](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10772)
Extend linkedevents app

## What was done
* Add new config variables
* Add card ghost
* Add loading animation
* Add bordered card variation

## How to test


* Check testing instructions here: https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/792
* Evaluate code quality

## Continuous documentation



## Translations



## Other PRs


[Etusivu](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/792)


[UHF-10772]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
